### PR TITLE
Add KvK is draw due to insufficient material

### PR DESCRIPTION
**Pull request by jjoshua2**
Step 1 of increasing adjudication abilities

### CODE REVIEW

File: src/Position.cpp [hunk 1]
```diff
@@ -905,7 +905,7 @@ Key Position::key_after(Move m) const {
 
 bool Position::is_draw() const {  //--didn't understand this _ply_ parameter; deleting it.
 
-  if (st->rule50 > 99 && (!checkers() || MoveList<LEGAL>(*this).size()))
+  if (st->rule50 > 99 && (!checkers() || MoveList<LEGAL>(*this).size()) || popcount(pos.pieces() == 2)
       return true;
```
Comment (Error323) [inline src/Position.cpp]: That should probably be `popcount(pos.pieces()) == 2`.

Comment (jjoshua2) [reply src/Position.cpp]: actually popcount(pos.pieces()) == 2)

Comment (Error323) [reply src/Position.cpp]: Yep :)

**Comment by jjoshua2** [comment]
Really messed up ) in my commits. Not sure why they were showing up in the wrong place. I think my edge browser isn't doing it right.

**Pull request closed without merging**